src/firebase/auth.js

```diff
@@ -17,7 +17,7 @@ export const doSignInWithEmailAndPassword = async (email, password) => {
     );
     return userCredential.user;
   } catch (error) {
-    throw error;
+    console.error("Error signing in with email and password:", error);
```

Comment:
We are expected not to have console logs in the committed code. It's fine to use them for debugging during dev, but it should not make it to the PR.
If what you need is the user to have visibility of the errors, then they should be displayed in the UI.